MediaGalleryRenditions/Test/Integration/Model/GetRenditionPathTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryRenditions\Test\Integration\Model;
+
+use Magento\Framework\App\Filesystem\DirectoryList;
+use Magento\Framework\Filesystem;
+use Magento\Framework\Filesystem\Directory\WriteInterface;
+use Magento\Framework\Filesystem\Directory\ReadInterface;
+use Magento\MediaGalleryRenditionsApi\Api\GetRenditionPathInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+class GetRenditionPathTest extends TestCase
+{
+    private const MEDIUM_SIZE_IMAGE = '/magento_medium_image.jpg';
+
+    private const LARGE_SIZE_IMAGE = '/magento_large_image.jpg';
+
+    /**
+     * @var GetRenditionPathInterface
+     */
+    private $getRenditionPath;
+
+    /**
+     * @var ReadInterface
+     */
+    protected static $mediaDir;
```

Comment:
Please avoid using static properties, it would be safer to ensure the test has a clear state each time it is run